src/routes/Apps/App/TrainDialogs.tsx

```diff
@@ -1071,6 +1071,21 @@ class TrainDialogs extends React.Component<Props, ComponentState> {
                 this.props.trainDialogReplayThunkAsync as any,
             )
 
+            // Get new activities to check for errors or warnings
+            const teachWithActivities = ((await this.props.fetchActivitiesThunkAsync(this.props.app.appId, newTrainDialog, this.props.user.name, this.props.user.id) as any) as Promise<CLM.TeachWithActivities>)
+            const replayError = DialogUtils.getMostSevereReplayError((await teachWithActivities).activities)
```

Comment:
It looks like you have incorrect `await` statement due to wrong types above.  Either way two `await`s is weird and wouldn't recommend this syntax in inline `await`s

Reply:
Ugh.. Sloppy coding. thx

Reply:
Wow. It's vscode that stuck that in.  I typed "a", TAB for auto-complete, and....

![vccode](https://user-images.githubusercontent.com/6528486/70284127-d3ca0280-1777-11ea-810f-48ef6d2d505b.gif)